lsst/ts/cbp/statemachine.py

```diff
@@ -231,159 +237,28 @@ def __init__(self, port: str, address: int, frequency: float = 2, initial_state:
         """
         self.model._cbp.set_park()
 
+    async def end_start(self, id_data):
+        self.model.connect()
+        self.telemetry_task = asyncio.ensure_future(self.telemetry())
 
-class CBPDeveloperRemote:
-    """This the class that provides an example of how to use the remote functionality for the CBP CSC.
+    async def end_standby(self, id_data):
+        self.telemetry_task.set_result('done')
+        self.model.disconnect()
```

Comment:
Consider putting the connect and disconnect code in `report_state` and base it on the state being reported. This offers the advantage that you can handle multiple states without worrying about how you got into them. In particular:

* You might wish to disconnect when going into FAULT, STANDBY or OFFLINE (if presently connected). If so, this can be done in one place instead of overriding three different state transition commands.
* If you connect when going into DISABLED or ENABLED (if not already connected). This allows you to write unit tests with an initial summary state of DISABLED or ENABLED, which saves the hassle of sending the state transition commands to go from STANDBY.